README.md

```diff
@@ -729,97 +731,144 @@ We have a small collection of algorithms, one for every day of the month. Scroll
+
+
+  - **Examples #2:**
+    ```
+    Input : 7
+    Output : 21
+    ```
+    **Explanation:**
+    - Tharun & his 6 friends come together to play this game. So to connect all of 7 cups, we need 28 strings.
+
+
+  - **Examples #3:**
+      ```
+      Input : 10
+      Output : 45
+      ```
+
+
+  - **Uses:**
     - [Triangular Series](https://en.wikipedia.org/wiki/Triangular_number)
     - [Networking between computers - Fully connected Mesh Topology](https://en.wikipedia.org/wiki/Network_topology)
     - [No. of greetings exchanged among 'n' people](https://www.quora.com/What-is-the-practical-use-of-triangular-numbers-in-the-real-world)
     - No. of matches played by 'n' teams
 
 
-## **December 30 - Number of Diagonals**
+### **December 30 - Number of Diagonals**
   - **Problem :**
     - There is a n-sided polygon. Find the number of [diagonals](https://en.wikipedia.org/wiki/Diagonal) that are possible. Do not count duplicates lines. Diagonals are Line segments that connect the non-adjacent vertices.
 
-  - **Example :**
-    1) Input : 4 (Quadrilateral - Rectangle)\
-       Output : 2
 
-       <img src="/src/docs/dec30-diagonals-of-rectangle.jpg" width="300" height="300"> <img src="/src/docs/dec30-pentagon.png" width="300" height="300">
+  - **Example #1:**
+    ```
+    Input : 4 (Quadrilateral - Rectangle)
+    Output : 2
+    ```
 
-    2) Input : 5 (Pentagon)\
-       Output : 5
+       <img src="/src/docs/Dec30-ExampleImg.jpg" width=483>
 
-    3) Input : 8\
-       Output : 20
 
-    4) Input : 11\
-       Output : 44 
+  - **Example #2:**
+    ```
+    Input : 5 (Pentagon)
+    Output : 5
+    ```
+
+  - **Example #3:**
+    ```
+    Input : 8
+    Output : 20
+    ```
+
+  - **Example #4:**
+    ```
+    Input : 11
+    Output : 44
+    ```
 
   - **Uses :**
     - [Graphics & Animation](https://en.wikipedia.org/wiki/B%C3%A9zier_surface)
     - [CAD](https://en.wikipedia.org/wiki/Computer-aided_design) and [Structural Engineering](https://en.wikipedia.org/wiki/Structural_engineering)
 
+
   - **Resources :**
-    - [Line segments and Points](https://www.basic-mathematics.com/number-of-segments.html) 
+    - [Line segments and Points](https://www.basic-mathematics.com/number-of-segments.html)
+
+
+### **December 31 - Closest Cell**
+  - **Problem**
+    - ClosestCell2(strArr) reads a 2D matrix that contains only 1, 0, or 2. From the index where a ‘1’ is, return the number of places either left, right, down, or up you must move to reach a cell which is represented by a 2. You can fold or wrap around the matrix to move to the cell which has ‘2’.
+      - The 2D matrix can have any number of 0's and 2's, but only a single 1.
+      -	It may not contain any 2's at all, then your program should return a 0.
+      -	You can try moving in each direction from the ‘1’, but you may need to add a special case if you reach the border and to jump to the other side of the matrix.
+    - Credits to coderbyte.com
+
+
+  - **Example**
```

Comment:
Can this be formatted better?
Just say 

Input: 
0 0 0
1 0 0
2 0 0

Output: 1
Explanation: It takes exactly one movement to reach 2.

Same for the next thing. 

Current format seems a bit weird.